ports/renesas-ra/Makefile

```diff
@@ -51,6 +51,16 @@ CMSIS_MCU_CAP = RA6M2
 USE_FSP_LPM = 1
 endif
 
+ifeq ($(BOARD),VK_RA6M5)
```

Comment:
I think all of those checks in the Makefile (like `ifeq  ($(BOARD), some_EK)`) should all be removed to simplify the Makefile, all the variables are already there:
```
BOARD_LOW  = (shell echo $(BOARD) | tr '[:upper:]' '[:lower:]')
CMSIS_MCU_CAP = $(CMSIS_MCU)
CMSIS_MCU_LOWER = $(shell echo $(CMSIS_MCU) | tr '[:upper:]' '[:lower:]')
USE_FSP_LPM ?=0 (or 1) // This should be moved to `mpconfigboard.mk` of each board.
```
Note there's a typo in the clicker board:
```
ifeq ($(BOARD),RA4M1_CLICKER)
BOARD_LOW = ra4m1_ek <- Should be ra4m1_clicker
```

Reply:
Well the design of the Makefile is what it is, I am just following it to integrate the VK boards. I don't know if I have the right to alter it. I think Mr. @TakeoTakahashi2020 has the last word here. Ask him why that is.

Reply:
Great idea, I am welcome to simplify Makefile. 
Regarding RA4M1_CLICKER's typo,  because it is not official EK board and not included in FSP, EK board's file is just referred instead.  However it's same situation as VK board and it should be solved by same way of VK boards.  I can check EK board and Clicker together, so please change as you propose.

Reply:
@TakeoTakahashi2020 I see the issue with `RA4M1_CLICKER` board, thanks for the insight. Even though `RA4M1_CLICKER` seems to build without linking `board_init.c` and `board_leds.c`, but those sources can still be added like so:

```Makefile
BOARD_LOW ?= $(shell echo $(BOARD) | tr '[:upper:]' '[:lower:]')
```

Then in `boards/RA4M1_CLICKER/mpconfigboard.mk` you can just set

```Makefile
BOARD_LOW=ra4m1_ek
```

I'd suggest a more descriptive name, like `FSP_BOARD_NAME`

Reply:
@iabdalkader , Thanks for your great review and suggetion.  I did build check quickly.  Yes, you are right.  The build is successful without board_init.c and board_leds.c.  I also think FSP_BOARD_NAME makes sense.
I completely agree with your idea.  I am happy to be able to simplify. Thanks.